### ISSUE TITLE
[Bug]: Unblock a user on a mobile device

### ISSUE THREAD

**Issue by fiXedTM** [nextcloud/server#35630]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

When you try to unlock a user from a mobile device (such as a phone), the dialog box disappears when clicked, and the user is still locked.

### Steps to reproduce

1. Log in to NextCloud (on your mobile device)
2. Click on your avatar in the upper right corner
3. Go to the "Users" option
4. Click the 3 dots next to the user
5. Try selecting the "Unblock User" option

### Expected behavior

After completing all the steps, I expected to unlock the user

### Installation method

Community Web installer on a VPS or web space

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.1

### Web server

Apache (supported)

### Database engine version

MySQL

### Is this bug present after an update or on a fresh install?

Fresh Nextcloud Server install

### Are you using the Nextcloud Server Encryption module?

Encryption is Enabled

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### List of activated Apps

```shell
Enabled:
  - circles: 25.0.0
  - cloud_federation_api: 1.8.0
  - comments: 1.15.0
  - contactsinteraction: 1.6.0
  - dashboard: 7.5.0
  - dav: 1.24.0
  - encryption: 2.13.0
  - federatedfilesharing: 1.15.0
  - federation: 1.15.0
  - files: 1.20.1
  - files_pdfviewer: 2.6.0
  - files_rightclick: 1.4.0
  - files_sharing: 1.17.0
  - files_trashbin: 1.15.0
  - files_versions: 1.18.0
  - firstrunwizard: 2.14.0
  - logreader: 2.10.0
  - lookup_server_connector: 1.13.0
  - nextcloud_announcements: 1.14.0
  - notifications: 2.13.1
  - oauth2: 1.13.0
  - onlyoffice: 7.6.8
  - password_policy: 1.15.0
  - privacy: 1.9.0
  - provisioning_api: 1.15.0
  - recommendations: 1.4.0
  - related_resources: 1.0.3
  - serverinfo: 1.15.0
  - settings: 1.7.0
  - support: 1.8.0
  - survey_client: 1.13.0
  - systemtags: 1.15.0
  - text: 3.6.0
  - theming: 2.0.1
  - twofactor_backupcodes: 1.14.0
  - updatenotification: 1.15.0
  - user_status: 1.5.0
  - viewer: 1.9.0
  - weather_status: 1.5.0
  - workflowengine: 2.7.0
Disabled:
  - activity: 2.16.0
  - admin_audit
  - bruteforcesettings: 2.4.0
  - calendar: 4.1.0
  - contacts: 5.0.1
  - files_external
  - impersonate: 1.11.0
  - mail: 2.1.1
  - photos: 2.0.0
  - richdocuments: 7.0.1
  - richdocumentscode_arm64: 22.5.802
  - sharebymail: 1.14.0
  - spreed: 15.0.1
  - suspicious_login
  - twofactor_totp
  - user_ldap
```

**Comment by szaimen** [comment]
Which nc version?

**Comment by ttv-fixed** [comment]
version is 25.0.1

**Comment by ttv-fixed** [comment]
After the last update, my problem still hasn't been fixed!

**Comment by fiXedTM** [comment]
@ref

**Comment by joshtrichards** [comment]
Duplicate of #21260
Consolidating tracking there.
Subscribe to that issue and/or give it a :+1: up if impacted.
